tests/gtest/test_population_Unfished_Initial.cpp

```diff
@@ -16,20 +16,24 @@ namespace
 
                 std::vector<double> mortality_F(nyears * nages, 0);
                 for (int fleet_index = 0; fleet_index < population.nfleets; fleet_index++)
-                {
+                { 
+                    if(!population.fleets[fleet_index]->is_survey){
                     size_t index_yf = year * population.nfleets + fleet_index;
                     mortality_F[index_ya] += population.fleets[fleet_index]->Fmort[year] *
```

Comment:
Suggest: add comments to explain where known values were used to test CalculateMortality and where FIMS code were used for comparison.

Reply:
comments have been added.